study-builder/src/main/java/org/broadinstitute/ddp/studybuilder/EventBuilder.java

```diff
@@ -250,7 +256,17 @@ private String actionAsStr(Config actionCfg) {
         } else if (EventActionType.COPY_ANSWER.name().equals(type)) {
             String copySourceQuestionStableId = actionCfg.getString("copySourceQuestionStableId");
             CopyAnswerTarget copyTarget = actionCfg.getEnum(CopyAnswerTarget.class, "copyTarget");
-            return String.format("COPY_ANSWER/%s/%s", copySourceQuestionStableId, copyTarget);
+            return String.format("%s/%s/%s", type, copySourceQuestionStableId, copyTarget);
```

Comment:
I understand the code was like this when you got here, but the team should pick a string format solution and use it all over the place. I think format(%s) is not used as much. Personally speaking, it can be hard to read, because you have to line up the var args with the format tokens.